auth_api_key/models/auth_api_key.py

```diff
@@ -25,9 +25,15 @@ def _retrieve_uid_from_api_key(self, api_key):
             ):
                 if not consteq(api_key, serv_config.get(section, "key")):
                     continue
+                user_env = self.env["res.users"]
+                if serv_config.has_option(section, "allow_inactive_user"):
+                    allow_inactive_user = serv_config.get(
+                        section, "allow_inactive_user")
+                    if allow_inactive_user not in ['False', 'false', False]:
+                        user_env = user_env.with_context(active_test=False)
```

Comment:
ConfigParser has [getboolean](https://docs.python.org/2/library/configparser.html#ConfigParser.RawConfigParser.getboolean), which might be useful.